helm/hpcc/values.yaml

```diff
@@ -34,6 +34,15 @@ global:
     resourceAttributes: # used to declare OTEL Resource Attribute config values
       deploymentEnvironment: development # used to anotate tracing spans' environment identifier (development/production/statiging/etc)
 
+  analyzerOptions:
```

Comment:
Are these excluded from the hash to restart the process?

Reply:
no - Shamser can you add to globalExcludeList (in hpcc.getConfigSHA)